playbooks/bootstrap.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Check remote connections
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  gather_facts: false
+  tasks:
+    - name: Wait 300 seconds, but only start checking after 10 seconds
+      ansible.builtin.wait_for_connection:
+        delay: 10
```

Comment:
I believe this should be parametrized, ideally in a way that allow you to skip this check if you want, or maybe set the time to be slightly long or shorter.